pkg/k8s/deployments/translate.go

```diff
@@ -219,6 +220,29 @@ func TranslateDevContainer(c *apiv1.Container, rule *model.TranslationRule) {
 	TranslateContainerSecurityContext(c, rule.SecurityContext)
 }
 
+func TranslateInitContainer(initContainer *model.InitContainer) {
+	if initContainer.Image == "" {
+		initContainer.Image = "okteto/bin:1.2.21"
```

Comment:
Can we put this in a const, so it's easier to find/change when we update the tag?

Reply:
we might even want to do the same we do for the Version, and make it injectable via our CI/CD process.

Reply:
@jLopezbarb can we use `model. OktetoBinImageTag` here?